app/src/main/kotlin/no/nav/tiltakspenger/vedtak/repository/søknad/SøknadDAO.kt

```diff
@@ -102,17 +126,19 @@ internal object SøknadDAO {
      */
     fun lagreHeleSøknaden(
         søknad: Søknad,
+        sakId: SakId,
```

Comment:
Hva om vi legger inn required sakId og saksnummer i Søknad.kt? Kan gjerne også ha sak_id i søknadstabellen, mens saksnummer kan joines inn i SELECT (typisk mønster i en del tabeller vi har)

Reply:
sak_id ligger allerede i søknadstabellen, så hvis jeg kan legge det på objektet så gjør det egentlig verden mindre forvirrende for meg 😅 Men, vil det kunne skape krøll at saken inneholder en liste med søknader hvis søknaden har en kobling til sak? Eller går det fint siden vi uansett legger til søknaden i listen på saken etter at søknaden er lagt til?